### DISCUSSION TITLE
Please take a look at pending review of project Qubic

### DISCUSSION THREAD

**Discussion by roadmanfong** [WalletConnect/walletconnect-monorepo#1915] [Developer Support]
Hi please take a look of pending review of Project Qubic
uuid: 8abaefe2-974c-4eef-ab57-ee4cc8ae8d3c

thanks

**Comment by yuquan08211** [comment]
Is this problem solved?
I also encountered the same problem。
pending review....

**Reply by roadmanfong** [reply]
No, it still pending

**Comment by roadmanfong** [comment]
Hi @bkrem 
Can you help review of this pending project.

Thank you
